library/build.gradle

```diff
@@ -26,33 +26,44 @@ jar {
     }
 }
```

Comment:
Also! are there any plans to have the library available on Maven Central as well? alongside the artifact in GitHub releases 📦